users-service/src/controllers/UsuarioController.js

```diff
@@ -1,46 +1,38 @@
-const Usuario = require('../models/Usuario');
+const UsuarioBusiness = require('../business/UsuarioBusiness');
 
 module.exports = {
    async findUser(request, response) {
       const { id } = request.headers;
 
-      const usuario = await Usuario.findById(id);
+      const usuario = await UsuarioBusiness.findUserById(id);
 
-      try {
-         if (!usuario) return response.status(200).json(usuario);
-         return response.status(400).json({ error: 'Usuario não existe' });
-      } catch (error) {
-         return response.status(400).json({ error: 'Usuario não existe' });
-      }
+      if (!usuario) return response.status(200).json(usuario);
+      return response.status(400).json({ error: 'Usuario não existe' });
```

Comment:
Status code de not found é 404

```suggestion
      return response.status(404).json({ error: 'Usuario não existe' });
```